syncd/syncd_flex_counter.cpp

```diff
@@ -799,13 +799,13 @@ void FlexCounter::collectCounters(
         status = sai_metadata_sai_buffer_api->get_ingress_priority_group_stats(
                         priorityGroupId,
                         static_cast<uint32_t>(priorityGroupCounterIds.size()),
-                        priorityGroupCounterIds.data(),
+                        (const sai_stat_id_t *)priorityGroupCounterIds.data(),
```

Comment:
from my perspective it would be better to have each stats accepting separate stat's enum but i guess this is already changed on the SAI api's

Reply:
Yes, SAI APIs were changed, so I had to do this cast.